lib/src/main/java/io/ably/lib/http/AnySyncHttp.java

```diff
@@ -0,0 +1,65 @@
+package io.ably.lib.http;
+
+import io.ably.lib.types.AblyException;
+import io.ably.lib.types.Callback;
+import io.ably.lib.types.ErrorInfo;
+
+/**
+ * Created by tcard on 28/04/2017.
+ */
+
+public class AnySyncHttp {
+    private final AsyncHttp asyncHttp;
+    private final SyncHttp syncHttp;
+
+    public AnySyncHttp(AsyncHttp asyncHttp, SyncHttp syncHttp) {
+        this.asyncHttp = asyncHttp;
+        this.syncHttp = syncHttp;
+    }
+
+    public static class Request<Result> {
```

Comment:
I don't see why this is `static` and yet it has a `anySyncHttp` member. I think you can remove that member and remove the `static` declaration.

Reply:
Done at 08f3c98.